### DISCUSSION TITLE
Feature request: Upload your own text

### DISCUSSION THREAD

**Discussion by ohong** [monkeytypegame/monkeytype#1734] [Ideas] [answered]
Hi MonkeyType team! Your current database of selected text is great, but I'd love to be able to upload my own text / passages for typing tests. Would you consider adding that as a feature? Thanks!

**Comment by ohong** [comment, marked as answer]
Actually, I just saw that there was a "Custom" feature. This is great! QQ: is there anyway to save a set of custom passages and rotate through them?

**Reply by Miodec** [reply]
Not currently, #1728 sounds like something simmilar though
